msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -42,19 +42,21 @@ Tabs allow the team members to access services and content in a specific space w
 
 With the pre-meeting app experience, you can find and add meeting apps. You can also do pre-meeting tasks, such as developing a poll to survey the meeting participants.
 
-To add tabs to an existing meeting:
+**To add tabs to an existing meeting**
 
 1. In your calendar, select a meeting to which you want to add a tab.
 1. Select the **Details** tab and select <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>. The tab gallery appears.
 
-    :::image type="content" source="~/assets/images/apps-in-meetings/Pre-Meeting-002.png" alt-text="Pre-meeting app experience.":::
+    :::image type="content" source="~/assets/images/apps-in-meetings/PreMeeting.png" alt-text="Pre-meeting app experience.":::
 
 1. In the tab gallery, select the app that you want to add and follow the steps as required. The app is installed as a tab.
 
    > [!NOTE]
    >
    > * You can also add a tab to an existing meeting using the meeting **Chat** tab.
-   > * Tab layout must be in an organized state, if there are more than 10 polls or surveys.
+   > * Install apps for the whole team by adding an app to channel meeting.
+   > * Add bots at team level if apps support.
```

Comment:
>   > * Add bots at team level if apps support.

What does this mean?

Reply:
@aosolis, would changing line #57 & #58 as "Adding an app to channel meeting installs the app for whole team scope. If the app supports bot, that will also be added at team level." would make more sense?!

Reply:
Let's add a new L3 heading in this section "Apps in channel meetings" or something like that.

Change this text to be
> A channel meeting has the same list of apps as its parent team. Installing an app to a channel meeting also makes it available in the parent team, and vice versa.
> 
> However, the tab instances in a channel meeting are separate from the tabs in the channel itself. For example, suppose a "Development" channel has a "Polly" tab. If we create a "Standup" meeting in that channel, that meeting would not have a "Polly" tab, until we explicitly [add it to the meeting](link to "To add tabs to an existing meeting").

Then follow it with the text that starts with "In scheduled public channel meetings, after a meeting app is added", which is a bit odd in its current location. Change the text "meeting app"  in that section to "tab" (I thinkthis is a case where we're loose with the distinction of app vs tab)

Reply:
Updated as mentioned.